src/components/Shifting/ShiftingBoard.tsx

```diff
@@ -0,0 +1,183 @@
+import PageTitle from "@/components/Common/PageTitle";
+import ShiftingBlock from "./ShiftingBlock";
+
+export default function BoardView() {
+  const { qParams, updateQuery, FilterBadges, advancedFilter } = useFilters({
+    limit: -1,
+    cacheBlacklist: ["patient_name"],
+  });
+
+  const [modalFor, setModalFor] = useState<ShiftingModel>();
+
+  const handleTransferComplete = async (shift?: ShiftingModel) => {
+    if (!shift) return;
+    await request(routes.completeTransfer, {
+      pathParams: { externalId: shift.external_id },
+    });
+    navigate(
+      `/facility/${shift.assigned_facility}/patient/${shift.patient}/consultation`,
+    );
+  };
+
+  const shiftStatusOptions = careConfig.wartimeShifting
+    ? SHIFTING_CHOICES_WARTIME
+    : SHIFTING_CHOICES_PEACETIME;
+
+  const COMPLETED = careConfig.wartimeShifting
+    ? [
+        "COMPLETED",
+        "REJECTED",
+        "CANCELLED",
+        "DESTINATION REJECTED",
+        "PATIENT EXPIRED",
+      ]
+    : ["CANCELLED", "PATIENT EXPIRED"];
+
+  const completedBoards = shiftStatusOptions.filter((option) =>
+    COMPLETED.includes(option.text),
+  );
+  const activeBoards = shiftStatusOptions.filter(
+    (option) => !COMPLETED.includes(option.text),
+  );
+
+  const [boardFilter, setBoardFilter] = useState(activeBoards);
+  const { t } = useTranslation();
+
+  return (
+    <div className="flex-col px-2 pb-2">
+      <div className="flex w-full flex-col items-center justify-between lg:flex-row">
+        <div className="w-1/3 lg:w-1/4">
+          <PageTitle
+            title={t("shifting")}
+            className="mx-3 md:mx-5"
+            hideBack
+            componentRight={
+              <ExportButton
+                action={async () => {
+                  const { data } = await request(routes.downloadShiftRequests, {
+                    query: { ...formatFilter(qParams), csv: true },
+                  });
+                  return data ?? null;
+                }}
+                filenamePrefix="shift_requests"
+              />
+            }
+            breadcrumbs={false}
+          />
+        </div>
+        <div className="flex w-full flex-col items-center justify-between gap-2 pt-2 xl:flex-row">
+          <SearchInput
+            name="patient_name"
+            value={qParams.patient_name}
+            onChange={(e) => updateQuery({ [e.name]: e.value })}
+            placeholder={t("search_patient")}
+          />
+
+          <Tabs
+            tabs={[
+              { text: t("active"), value: 0 },
+              { text: t("archived"), value: 1 },
+            ]}
+            onTabChange={(tab) =>
+              setBoardFilter(tab ? completedBoards : activeBoards)
+            }
+            currentTab={boardFilter[0].text !== activeBoards[0].text ? 1 : 0}
+          />
+
+          <div className="flex w-full flex-col gap-2 lg:mr-4 lg:w-fit lg:flex-row lg:gap-4">
+            <ButtonV2
+              className="py-[11px]"
```

Comment:
rem values preferred over arbitrary pixel values

Reply:
Seems like they were added for the buttons on the top right to have the same height. No sub of 11px in TW, so letting it be : (